samples/inmemory/Semanticcache.ipynb

```diff
@@ -0,0 +1,270 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "04ad9c80-2b3a-4df1-bead-fb347d51359f",
+   "metadata": {},
+   "source": [
+    "## 1. Set environment vairable for MemoryDB cluster "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "41f56fd0-efc1-4edb-9a99-d41de0954949",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "%env MEMORYDB_HOST=your_cluster_endpoint\n",
+    "%env MEMORYDB_PORT=6379"
```

Comment:
Suggestion to remove setting env variables in the notebook as the main reason to use env variables are so that they are set by the host, and they are not visible directly in the notebook. We can add a note that these env variables are expected to run this notebook. 
An alternative would be to just use regular variables to store these values.